tools/cmake_test.py

```diff
@@ -133,15 +133,15 @@ def _find_addr2lines(self):
         # Workstation: find our build directory by searching back from binary
         path_parts = self.binary.split("/")
         try:
-            vbuild = "/".join(path_parts[0:path_parts.index("vbuild") + 3])
+            vbuild = "/".join(path_parts[0:path_parts.index("vbuild") + 5])
```

Comment:
nit: It would be good to have an example path here is a comment to help future generations.